public/app/plugins/panel/canvas/editor/options.ts

```diff
@@ -39,6 +40,7 @@ export const optionBuilder: OptionSuppliers = {
         editor: ResourceDimensionEditor,
         settings: {
           resourceType: 'image',
+          defaultTab: PickerTabType.URL,
```

Comment:
🤔 I don't think we really want URL to be the default, just that if an URL is specified it is shown as the option

Reply:
In this case the URL is the default option only for the root background image for canvas. If we set logic to show URL tab based on if a url exists that messes up the resource picker in the context of picking icons as we set a url by default for any selected icon
<img width="316" alt="Screen Shot 2022-10-20 at 10 25 46 AM" src="https://user-images.githubusercontent.com/22381771/197017329-232d4d79-1067-4357-aef9-87ffbb250345.png">

Reply:
but the URL we set is one that is known to us right?  it is relative to our root and actually points to a folder we know about.

the URL will start with `http://...` i think